### ISSUE TITLE
not saved any changes

### ISSUE THREAD

**Issue by Nemra1** [Col-E/Recaf#48]
This my tester in last version
drag java file to recaf > select my class file > changes my code > save as new.jar
when i check my changes ..nothing happend and nothing changed

**Comment by Col-E** [comment]
Can you share your sample / what changes you made?

I can't think of what could cause this with the description you provided.

**Comment by Col-E** [comment]
This was most likely the same issue as https://github.com/Col-E/Recaf/issues/54

**Comment by zwb1988** [comment]
I downloaded and used Recaf 1.1.0 and getting this issue as well. I modified two class files in a jar package, but only the latter one got saved. I used the Save Jar button on the UI to save a new jar file.

**Comment by Col-E** [comment]
Well can you tell me exactly how you edited your file? I can't recreate this.

**Comment by zwb1988** [comment]
I open a jar file, navigated to a package, updated an instruction from ICONST_0 to ICONST_1 in a class and closed the tab. Then I opened another file in another package, modified a few methods there, then I close tab again. I hit the Save Jar button to save everything into a jar file. In the end, I found the class I updated with the ICONST instruction remained unchanged.

 I was only able to create this once and couldn't confirm this can happen every time.
